Service/defect/src/main/java/com/cityu/defect/controller/DefectInfoController.java

```diff
@@ -58,6 +58,10 @@ public BaseResponse<List<DefectInfoVO>> searchDefectInfoVO(@RequestBody DefectIn
         if(loginUser == null){
             throw new BusinessException(ErrorCode.NOT_LOGIN_ERROR);
         }
+        //需要传递userId
+        if(defectInfoQueryRequest.getUserId()==null){
+            throw new BusinessException(40001,"Your request have to contain your userId");
```

Comment:
Do not use constant numbers in your code. Write 40001 into ErrorCode file

Reply:
ok, my bad